examples/js/footer/footer-table.js

```diff
@@ -0,0 +1,67 @@
+/* eslint max-len: 0 */
+import React from 'react';
+import { BootstrapTable, TableHeaderColumn } from 'react-bootstrap-table';
+
+
+const products = [];
+
+function addProducts(quantity) {
+  const startId = products.length;
+  for (let i = 0; i < quantity; i++) {
+    const id = startId + i;
+    products.push({
+      id: id,
+      name: 'Item name ' + id,
+      price: 2100 + i
+    });
+  }
+}
+
+addProducts(70);
+
+export default class FooterTable extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const options = {
+      footerData: [
```

Comment:
@AllenFang You might want to take a look here, if passing footerData in `options` is a preferred way or it should be passed as a prop to `BootstrapTable`. Please suggest.

Reply:
@prajapati-parth sorry for lately reply....  I'll prefer on `BootstrapTable`